client/src/components/inputs/tests/__snapshots__/Field.test.js.snap

```diff
@@ -29,53 +29,66 @@ ShallowWrapper {
     "nodeType": "host",
     "props": Object {
       "children": Array [
-        "Email:",
-        <input
-          className="field normal"
-          error={Object {}}
-          id="email"
-          name="email"
-          onChange={[Function]}
-          placeholder="Enter email..."
-          type="email"
-          value=""
-        />,
+        <p
+          className="field-label"
+        >
```

Comment:
what's up with the new line, this is ugly

Reply:
hmm maybe not

Reply:
That’s the standard and I couldn’t figure out the prettier config change

Reply:
Also, this is a snapshot, so it is generated. I didn’t make this file. It is for unit testing